app-emulation/wine-vanilla/files/wine-vanilla-5.22-gcc11-sincos.patch

```diff
@@ -0,0 +1,87 @@
+https://bugs.gentoo.org/787665
+
+https://source.winehq.org/git/wine.git/commit/f0131276
```

Comment:
Up to you, but I would be totally fine if we only supported in-tree testing wine versions (7.5-7.8,9999) and left older versions unsupported

Reply:
Either way is fine for me, I really only care for latest :) Only did the rest so there wouldn't be disparity in the way mingw is handled (this is just about dropping commits, so just say which you prefer).

Reply:
lets keep this too since you did the work